x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/response/schemas.ts

```diff
@@ -25,6 +25,21 @@ export const file_name = t.string;
  */
 export const filters = t.array(t.unknown); // Filters are not easily type-able yet
 
+/**
+ * Params is an "object", since it is a type of AlertActionParams which is action templates.
+ * @see x-pack/plugins/alerting/common/alert.ts
+ */
+export const action = t.exact(
+  t.type({
+    group: t.string,
+    id: t.string,
+    actionTypeId: t.string,
```

Comment:
We might not have a choice but to keep this as camel case 🐫 , but if possible we should try for 🐍 snake case.

Up to you on this one, I know that the alerting team was made aware that they should be using snake case.

I leave this part as optional, but the more snake case we have now, the less migrations or headaches and support later for the API.